app/src/main/java/io/github/androidacademyglobal/ui/profile/ProfileScreen.kt

```diff
@@ -32,22 +32,22 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
-import androidx.lifecycle.compose.ExperimentalLifecycleComposeApi
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
 import io.github.androidacademyglobal.R
 import io.github.androidacademyglobal.ui.components.baselineHeight
 import io.github.androidacademyglobal.ui.theme.AppConstants
 import io.github.androidacademyglobal.ui.theme.DimensionConstants
 import org.koin.androidx.compose.getViewModel
 
-@OptIn(ExperimentalMaterial3Api::class, ExperimentalLifecycleComposeApi::class)
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun ProfileScreen(
     viewModel: ProfileViewModel = getViewModel()
 ) {
     val userData by viewModel.state.collectAsStateWithLifecycle()
     val topAppBarState = rememberTopAppBarState()
-    val scrollBehavior = remember { TopAppBarDefaults.pinnedScrollBehavior(topAppBarState) }
+    val pinnedScrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(topAppBarState)
+    val scrollBehavior = remember { pinnedScrollBehavior }
```

Comment:
calling inside `remember` is better, imho

Reply:
`TopAppBarDefaults.pinnedScrollBehavior` is a Composable and can not be called within `remember`. Actually, I think I'll rework this screen in the nearest commits.